### ISSUE TITLE
[WordPress] No image preview in media library for webp images

### ISSUE THREAD

**Issue by asol** [bitnami/vms#1147]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

d4d64396-44c1-e453-be26-d2762b80c729

### bndiagnostic output

    ? Resources: Found possible issues
    ? Connectivity: Found possible issues
    ✓ Mariadb: No issues found
    ✓ Processes: No issues found
    ✓ Wordpress: No issues found
    ? Apache: Found possible issues
    ✓ Php: No issues found

### bndiagnostic was not useful. Could you please tell us why?

not sure

### Describe your issue as much as you can

webp images have no preview in media library, can see them on the website but not in preview in wp-admin.

**Comment by jotamartos** [comment]
Hi @asol,

Thank you for using Bitnami. I can see there are some related tickets in the WordPress' forums. Please take a look at them and contact the WordPress' support team or the plugin's one (if any) to get more information

https://wordpress.org/support/topic/webp-is-not-generated-for-thumbnails/
https://wordpress.org/support/topic/some-thumbnails-not-showing-up-2/

**Comment by asol** [comment]
I tried both, no love.
I also tried to add rewrite code: https://gist.github.com/lukecav/fc5e26215cc5be5cf0d2b77212f3e7e0
to /opt/bitnami/apache/conf/vhosts/htaccess/wordpress-htaccess.conf file
no love

The site moved to bitnami from wordpress.com, something didn't move right in wordpress,com everything works perfectly.
Like all thumbnail missing.
Pictures in main website missing
When open the pic itself in media folder in wordpress the looks ok.
